### PR TITLE
Truncate day percentage numbers for non googlers

### PR DESCRIPTION
**Pull request by beaufortfrancois**
PR #185 has been reverted so here's a new version which truncates day percentage numbers for non googlers.

TBR=@ebidel
BUG=#123

### CODE REVIEW

File: common.py [hunk 1]
```diff
@@ -46,6 +46,22 @@ def __init__(self, request, response):
 
 class JSONHandler(BaseHandler):
 
+  def _is_google_corp_user(self, user):
```
Comment (ebidel) [inline common.py]: How about `_is_googler`

**Comment by ebidel** [comment]
Reviewed

**Comment by beaufortfrancois** [comment]
`s/is_google_corp_user/_is_googler/g` successful.

**Comment by ebidel** [comment]
:100:

**Pull request merged**